src/main/java/app/fitbuddy/controller/crud/HistoryController.java

```diff
@@ -77,21 +77,21 @@ public List<HistoryResponseDTO> readAll(@PathVariable("date") String strDate) {
 	}
 	
 	@PutMapping("{id}")
-	public void update(@PathVariable("id") Integer historyId, @Valid @RequestBody HistoryUpdateDTO historyUpdateDTO) {
+	public void update(@PathVariable("id") @NotNull Integer historyId, @Valid @RequestBody HistoryUpdateDTO historyUpdateDTO) {
 		Authentication auth = SecurityContextHolder.getContext().getAuthentication();
-		if (historyId != null) {
+		
 			Integer userId = appUserCrudService.readByName(auth.getName()).getId();
 			if (userId != null) {
 				historyCrudService.update(historyId, historyUpdateDTO);
 				logger.info("Updating history: {}", historyUpdateDTO);				
 			}
-		}
+		
```

Comment:
- Remove empty line in Line 82
- Fix indentation in Line 83 to 87
- Remove empty line in Line 88